internal/utils/kruize/kruize_api.go

```diff
@@ -84,7 +84,7 @@ func Create_kruize_experiments(experiment_name string, cluster_identifier string
 			return nil, fmt.Errorf("%s", resdata["message"])
 		}
 	}
-
+	createExperimentRequest.Inc()
```

Comment:
This will only increment if createExperiment request was successfully. To track all the request `createExperimentRequest.Inc()` should be on line 57.